getting-started-publishing/publishing-descriptions.md

```diff
@@ -1,15 +1,55 @@
 ---
 info_en: Page about "Getting started - Publishing descriptions"
-info_no: Side om "Komme i gang - Publisere beskrivelser/kataloger"
+info_no: Side om "Komme i gang - Publisere beskrivelser/kataloger via høsteadmin-grensesnittet"
 description: Skal gi overordnet informasjon om hvordan publisere beskrivelser/kataloger til data.norge.no
-url: https://data.norge.no/getting-started/publishing-data
-note: Vurder å slå sammen med "Creating descriptions"
+url: https://data.norge.no/getting-started/setup-harvesting
 ---
 
-# Publisere beskrivelser
+# Hvordan publisere beskrivelser
 
-## Guide
+Har du brukt registreringsløsningen til å lage beskrivelser, og publisert dem derifra, blir katalogen automatisk lagt til for høsting.
```

Comment:
Legge til noe om at du kan trigge en høsting manuelt fra høsteadmin, selv om du bruker registreringsløsningen. Gjerne også hvilke feilsituasjoner som kan løses med dette, om vi kjenner til noen.

Reply:
Jeg lurer også litt på om vi skulle delt opp i sette opp en høsting og trigge en høsting i to ulike sider.

Reply:
Bra forslag!